### PR TITLE
Add the rediss (ssl_redis) server and ssl keys in app.yaml.

### PR DESCRIPTION
**Pull request by koning**
Add the rediss server.
    Allow users to define ssl keys either as full paths or in the
    celery/certs dir.
    Add a url option to override the rest of the app.yaml broker or results
    backend config.
    Add function docstrings.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Bug that prevented an empty username for results backend and broker when using redis.
 - Bug that prevented `OUTPUT_PATH` from being an integer.
 
+### Added
+- Optional ssl files for the  broker and results beackend config.
```
Comment (lucpeterson) [inline CHANGELOG.md]: extra space

**Comment by ben-bay** [comment]
When I run them by hand, they seem to succeed. Looking further...

File: docs/source/modules/installation/installation.rst [hunk 2]
```diff
@@ -175,11 +175,20 @@ Some aliases can be defined for convenience.
   alias python3="docker exec my-merlin python3"
 
 When you are done with the containers you can stop them using ``docker-compose down``.
+We will be using the containers in the subsequent modules so leave them running.
```
Comment (lucpeterson) [inline docs/source/modules/installation/installation.rst]: this file change seems like it's from a different PR?

**Comment by koning** [comment]
Yes, I ran them and they passed as well. I put in a potential fix.

**Comment by koning** [comment]
Tests are passing now.

File: docs/source/modules/installation/installation.rst (hunk 2 above)
Comment (koning) [reply docs/source/modules/installation/installation.rst]: I added tutorial doc changes since tutorial was moved into develop.

**Pull request merged**